cypress/support/calendar.js

```diff
@@ -0,0 +1,112 @@
+import localforage from 'localforage';
+import { Link, MultiColumnListCell, Pane, Modal, Button } from '../../interactors';
+
+Cypress.Commands.add('openCalendarSettings', (isLoggedIn = true) => {
```

Comment:
you can use just

cy.loginAsAdmin({ path: ..., waiter: ...})

in berore or beforeEach code block